data/helpers.d/network

```diff
@@ -47,7 +47,7 @@ ynh_find_port () {
 # | arg: path - the web path to check the availability of
 ynh_webpath_available () {
 	local domain=$1
-	local path=$(ynh_normalize_url_path $2)
+	local path=$2
```

Comment:
I don't know, because if we keep a call to `ynh_normalize_url_path` in `ynh_webpath_register`, it's not needed to add it in the install script.
So, I think it's more simple for all the packagers like that.

And it avoids a forgot.